README.rst

```diff
@@ -802,6 +802,132 @@ List locations with devices deployed (in GeoJSON format)
 
     GET api/v1/controller/location/geojson/
 
+List devices with location
+^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+.. code:: text
+
+    GET api/v1/geo/devicelocation/
```

Comment:
@nemesisdesign with this pattern `/api/v1/controller/device/{pk}/location/` an endpoint is already present. should I edit it's view to make it for CRUD oerations.

Reply:
yes, definitely, right now we should have only GET, right? You can add support for PUT and DELETE.
Probably not POST because there can be only one device location per device.

Reply:
this has been achieved.